doxia-core/src/main/java/org/apache/maven/doxia/sink/impl/AbstractSink.java

```diff
@@ -23,12 +23,337 @@
 
 /**
  * An abstract base class that defines some convenience methods for sinks.
+ * Also acts as compatibility bridge for Doxia 1.0 methods which have overridden variants in Doxia > 1.0
```

Comment:
So this is temporary?

Reply:
As long as the methods not taking SinkEventAttributes are not deprecated, this is permanent.